### ISSUE TITLE
Human Walk Snipe Bug!!

### ISSUE THREAD

**Issue by d4nvjpl0ng4n** [NoxxDev/NecroBot#914]
[22:13:39] (SNIPER) (HUMAN WALK) Found Snorlax spawning | lat: 40.774341, lng: -
73.97123, dist 231.35m , expired in 14 min 39 sec | Estimate : 00 min 28 sec | A
llow spin: No | Allow catch : No | Speed 60.00km/h
[22:13:39] (INFO) Traveling to Pokestop: Memorial Stone (242m) (32 seconds) (rou
te GoogleWalk)
[22:14:18] (SNIPER) (HUMAN WALK) destination reached | lat: 40.774341, lng: -73.
97123 | wait :0.00 sec
[22:14:20] (INFO) Skipped Spearow
[22:14:20] (SNIPER) (HUMAN WALK) Found Electabuzz spawning | lat: 40.767054, lng
: -73.973682, dist 1275.26m , expired in 03 min 54 sec | Estimate : 01 min 31 se
c | Allow spin: Yes | Allow catch : No | Speed 60.00km/h
[22:14:21] (INFO) Traveling to Pokestop: Memorial Stone (1052m) (140 seconds) (r
oute GoogleWalk)
[22:17:28] (SNIPER) (HUMAN WALK) destination reached | lat: 40.767054, lng: -73.
973682 | wait :0.00 sec
[22:17:30] (SNIPER) (HUMAN WALK) Found Rhydon spawning | lat: 40.7639261791, lng
: -73.9818551019, dist 1193.32m , expired in 09 min 13 sec | Estimate : 01 min 2
6 sec | Allow spin: Yes | Allow catch : Yes | Speed 60.00km/h
[22:17:31] (INFO) Traveling to Pokestop: Memorial Stone (1138m) (151 seconds) (r
oute GoogleWalk)
[22:17:52] (INFO) Skipped NidoranMale
[22:17:52] (INFO) (CATCH LIMIT) 135/998
[22:17:53] (PKMN) (CatchMissed) | (Normal) Jigglypuff Lvl: 29 CP: (694/760) IV:
66.00% | Chance: 27.8% | 63.27m dist | with a PokeBall (8 left). | lat: 40.76797
6 long: -73.974509 | Move1: PoundFast Move2: BodySlam

-----------------------------------------------------------------------------------------------------------------
Necrobot found out the pkm to walk-snipe but when reached to the destination, It caught another pkm or go to other gym but not catching the desired pkm

Please help me

**Comment by kuntau** [comment]
I'm pretty sure that was a fake signal

**Comment by d4nvjpl0ng4n** [comment]
I sniped manually, it's really a true pkm. not fake

**Comment by hardknoxni69a** [comment]
i was going to say fake as well which is pretty sad how much time we waste going after fakes but alot of time they are really there as i have filled up quite a bit thanks to human sniping did you double check the location and make sure that u didnt snipe a different location?

**Comment by zyqsxxxzj** [comment]
I have the same problem too. Three days ago, this function worked well. But then the coordinates from PokeRadar and Skiplagged are both invalid. Only the coordinates from PokeSnipers works.

**Comment by pokeplayer2** [comment]
Hi, 
I think it works, but I never see that it speed up to 48 km/h:

  "HumanWalkSnipeConfig": {
    "DisplayPokemonList": true,
    "MaxDistance": 500.0,
    "MaxEstimateTime": 900.0,
    "CatchEmAllMinBalls": 30,
    "TryCatchEmAll": false,
    "CatchPokemonWhileWalking": false,
    "SpinWhileWalking": true,
    "AlwaysWalkback": false,
    "SnipingScanOffset": 0.025,
    "Enable": true,
    "WalkbackDistanceLimit": 300.0,
    "IncludeDefaultLocation": true,
    "UsePokeRadar": true,
    "UseSkiplagged": true,
    "UseSnipePokemonList": true,
    "MaxSpeedUpSpeed": 48.0,
    "AllowSpeedUp": true,
    "DelayTimeAtDestination": 10000

**Comment by hardknoxni69a** [comment]
i would probably set those to false for now check back and see when they start working again some people just like making us look like fools chasing down pokemon that arent there kind of like when a mew pops on on locationfeeder lol

**Comment by pokeplayer2** [comment]
Switch off only Skiplagged and it works fine for me at the moment in Berlin.

**Comment by pokeplayer2** [comment]
But still no speed up to 48 km/h ...

**Comment by samuraitruong** [comment]
Pokeradar giving wrong coord since sunday. Use fastpokemap instead

**Comment by kuntau** [comment]
Indeed, after they're down on sunday. Unfortunately only pokeradar available on SEA region
